src/Ushahidi/Contracts/Usecase.php

```diff
@@ -42,4 +42,28 @@ public function isSearch();
      * @return array
      */
     public function interact();
+
+    // /**
+    //  * Set parameters that can be used to uniquely identify a **single** entity:
+    //  *
+    //  * @param array $identifiers
+    //  * @return $this
+    //  */
+    // public function setIdentifiers(array $identifiers);
+
+    // /**
+    //  * Set parameters that can be used to uniquely identify a **single** entity:
+    //  *
+    //  * @param array $payload
+    //  * @return $this
+    //  */
+    // public function setPayload(array $payload);
```

Comment:
I would remove that instead of commenting out. We've got version control that we can use if we decided we need that piece of code.